packages/perspective-jupyterlab/src/ts/psp_widget.ts

```diff
@@ -18,6 +16,9 @@ import {MIME_TYPE, PSP_CLASS, PSP_CONTAINER_CLASS, PSP_CONTAINER_CLASS_DARK} fro
 
 import {HTMLPerspectiveViewerElement, Pivots, Aggregates, Sort, ComputedColumns, PerspectiveViewerOptions, Filters, Columns} from "@finos/perspective-viewer";
 
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const perspective = require("@finos/perspective");
```

Comment:
Webpack advises [not mixing `require` and `import`](https://stackoverflow.com/questions/56068126/is-it-possible-to-use-both-require-and-import-together-with-webpack) - is this _required_  because the TypeScript bindings are not correct or something?

Reply:
yes